src/ui/screens/getHelp/developerMenu/ViewImagesPlayground.qml

```diff
@@ -23,99 +25,127 @@ Item {
         _menuOnBackClicked: () => getHelpStackView.pop()
     }
 
+    Rectangle {
+        id: imageContainer
+
+        anchors.top: menu.bottom
+        color: MZTheme.colors.bgColor
+        height: 375
+        width: parent.width
+
+        MZInterLabel {
+            // Do not translate this string!
+            text: "No image selected"
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+            color: MZTheme.colors.fontColor
+            visible: imageSelect.currentIndex < 0
+        }
+        
+        ColumnLayout {
+          spacing: 10
+          anchors.centerIn: parent
+          RowLayout {
+              spacing: 50
+              // anchors.centerIn: parent
+              Text {
+                text: "Asset for light themes"
+                color: MZTheme.colors.fontColor
+                visible: imageSelect.currentIndex >= 0
+              }
+
+              Text {
+                text: "Asset for dark themes"
+                color: MZTheme.colors.fontColor
+                visible: imageSelect.currentIndex >= 0
+              }
+          }
+          RowLayout {
+              spacing: 25
+              Image {
+                  source: selectedImageSourceLight
+                  sourceSize.height: 150
+                  sourceSize.width: 150
+                  fillMode: Image.PreserveAspectFit
+              }
+
+              Image {
+                  source: selectedImageSourceDark
+                  sourceSize.height: 150
+                  sourceSize.width: 150
+                  fillMode: Image.PreserveAspectFit
+              }
+          }
+          
+            RowLayout {
+              spacing: 25
+              Rectangle {
+                height: 150
```

Comment:
Weird QML quirk/gotcha: QML components that are inside ("children") of Layout components (RowLayout, ColumnLayout, AnythingLayout) need to use a different sizing rules. Not doing so can cause truly bizarre, completely unexpected behavior, a lot of console complaining, and even crashes. In this case `height` should be `Layout.preferredHeight` and `width` should be `Layout.preferredWidth`. Same thing for the height/width rules on L96-97.

Reply:
Updated, thanks